ena-submission/src/ena_deposition/create_sample.py

```diff
@@ -143,13 +143,42 @@ def construct_sample_set_object(
     return SampleSetType(sample=[sample_type])
 
 
+def set_sample_table_entry(db_config, row, seq_key):
```

Comment:
Same, here, good to check for sample's existence against ENA API like this: https://www.ebi.ac.uk/ena/browser/view/PRJEB402